ui/lib/pki/addon/components/page/pki-configuration-details.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import errorMessage from 'vault/utils/error-message';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+//TYPES
+import RouterService from '@ember/routing/router-service';
+import FlashMessageService from 'vault/services/flash-messages';
+import Store from '@ember-data/store';
+
+interface Args {
+  currentPath: string;
+}
+
+export default class PkiConfigurationDetails extends Component<Args> {
+  @service declare readonly store: Store;
+  @service declare readonly router: RouterService;
+  @service declare readonly flashMessages: FlashMessageService;
+
+  @tracked showDeleteAllIssuers = false;
+
+  @action
+  async deleteAllIssuers() {
+    try {
+      const issuerAdapter = this.store.adapterFor('pki/issuer');
+      issuerAdapter.deleteAllIssuers(this.args.currentPath);
```

Comment:
I wonder if we should `await` the adapter method here 🤔

Reply:
good question! I `console.log` issuerAdapter with and without await and they print out the same thing. since issuerAdapter isn't a promise, I don't think we need to `await`

Reply:
Okay! I was asking since `deleteAllIssuers` is an ajax request, right? I was wondering if a user has a lot of issuers, then the delete operation might take a moment.

Reply:
Good point! i'll add the await!